global/scripts/golangci-lint/.golangci.yml

```diff
@@ -1,8 +1,351 @@
+# This code is licensed under the terms of the MIT license https://opensource.org/license/mit
```

Comment:
Could you refer the link original (https://gist.github.com/maratori/47a4d00457a92aa426dbd48a18776322) in this file as comment? We might want to check times to times what are the differences and updates.

Reply:
Sure.

Reply:
Done.